public/app/core/utils/regionalizeText.ts

```diff
@@ -0,0 +1,26 @@
+const TRANSLATIONS_MAP: Record<string, Record<string, string>> = {
+  'en-US': {
+    ising: 'izing',
+  },
+  default: {
+    izing: 'ising',
+  },
+};
+
+export function regionalizeText(input: string): string {
+  const userLanguage = navigator.language;
+  const translationsMap = TRANSLATIONS_MAP[userLanguage] || TRANSLATIONS_MAP['default'];
+
+  // look through translations available for user language,
+  // return modified string if match found.
+  for (let preTranslation in translationsMap) {
+    if (input.endsWith(preTranslation)) {
+      let baseIdx = input.length - preTranslation.length;
+      let postTranslation = translationsMap[preTranslation];
+      return input.substring(0, baseIdx) + postTranslation;
+    }
+  }
+
+  // if no translation found, simply return original input.
+  return input;
+}
```

Comment:
@conorevans thanks a lot for your first contribution, we really appreciate it! Good code and excellent that you have added tests for this function. My major concern tho is that it's quite rigid, meaning that it will only apply for texts that end with specified strings defined in mappings.

Since i18n is a complex problem that we haven't solved and don't think we gonna tackle very soon, I suggest a very simple approach to this particular issue that you are trying to solve, and just do `navigator.language === 'en-US' ?  'Democratizing data' :  'Democratising data';`

Reply:
Hey @dprokop that's totally fair. I've now got a few contributions past the post but yeah I guess since it would have been my first one, I thought it best to take a holistic approach in terms of the function/testing. It is indeed somewhat rigid - as I mentioned in description and as you note, a full i18n solution is beyond the scope for this issue, but I possibly should have done by `sed`ing beforehand to find that there's only one occurrence of an `-ise/-ize` in the public facing content 😅 - a simple if/else logic, as you suggested, seems more appropriate alright. I'll update that now.